cmd/ocitools/validate_test.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"testing"
+
+	rspec "github.com/opencontainers/runtime-spec/specs-go"
+)
+
+func TestCheckSemVer(t *testing.T) {
+	var spec rspec.Spec
+	cases := []struct {
+		v        string
+		expected int
+	}{
+		{rspec.Version, 0},
+		{"0.0.1", 1},
+		{"invalid", 2},
+	}
+
+	for _, c := range cases {
+		spec.Version = c.v
+		msg := checkSemVer(spec, "", false)
+		if len(msg) != c.expected {
```

Comment:
Is “number of error messages” really what we want to be checking here?  I'd rather have a bunch of potential version strings with true/false for whether we thought they were valid, and error if we thought a version was valid and `checkSemVer` gave us error messages at all (logging the error messages with `t.Fatalf`) or if we thought a version was invalid and `checkSemVer` did not give us error messages.